cinema/serializers.py

```diff
@@ -44,14 +44,31 @@ class MovieListSerializer(MovieSerializer):
         many=True, read_only=True, slug_field="full_name"
     )
 
+    class Meta(MovieSerializer.Meta):
+        fields = MovieSerializer.Meta.fields + ("image", )
```

Comment:
maybe you can just add field image in base MovieSerializer

Reply:
no, then test didn't pass, because we shuld can add image only on detail page